susApp/www/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
```

Comment:
A general comment on controllers. If the function (e.g $scope.[functionName]) is not being called on html OR is only called within the controller file, make it a private function. This will help performance of the overall app Eg.

```
var functionName = function () {
// your code here 
}
```